internal/pkg/template/template_functions.go

```diff
@@ -140,6 +140,12 @@ func generateMountPointJSON(mountPoints []*MountPoint) string {
 
 }
 
+// PortMappingName returns name for the port mapping.
+func PortMappingName(containerName string, containerPort string) string {
+	fmt.Println("I was here ", containerName+containerPort)
+	return containerName + containerPort
```

Comment:
Is it okay according to the ECS API to have digits in the name of a port?

Reply:
yes it is allowed - The characters can include lowercase letters, numbers, underscores (_), and hyphens (-). A hyphen can't be the first character.